src/util/apis/generated/models/genericLog.js

```diff
@@ -144,11 +164,25 @@ GenericLog.prototype.mapper = function () {
             name: 'String'
           }
         },
+        messageId: {
+          required: false,
+          serializedName: 'message_id',
+          type: {
+            name: 'String'
+          }
+        },
         properties: {
           required: false,
           serializedName: 'properties',
           type: {
-            name: 'String'
```

Comment:
found the change, is that propertyName a fancy writing methods to indicate the key?